resources/views/organisations/show.blade.php

```diff
@@ -6,7 +6,7 @@
         @if(! $organisation->agreement_signed_at && $organisation->admins->contains(auth()->user()))
         <div class="alert alert-warning show text-dark my-3">Before continuing, please check and agree to the data sharing agreement, available on the Settings tab.</div>
         @elseif(! $organisation->agreement_signed_at && !$organisation->admins->contains(auth()->user()))
-        <div class="alert alert-warning text-dark my-3">This institution has not yet digitally signed the data sharing agreement. An institution admin should update the settings.</div>
+        <div class="alert alert-warning text-dark my-3">This institution has not yet agreed to the data sharing agreement. An institution admin should update the settings.</div>
```

Comment:
This hotfix branch came from `main`, so this change is one that was made on main as a previous hotfix and not yet merged into dev. The "agreed to" text is correct.